funnel/views/index.py

```diff
@@ -1,22 +1,33 @@
 # -*- coding: utf-8 -*-
 
 from flask import g, render_template, redirect, jsonify
-from datetime import datetime
-from coaster.views import jsonp, load_model
-from .. import app
+from coaster.views import jsonp, load_model, render_with
+from .. import app, funnelapp
 from ..models import Profile, ProposalSpace, Proposal
 from .space import space_data
 
 
 @app.route('/')
+@render_with({'text/html': 'index.html.jinja2'}, json=True)
 def index():
     g.profile = None
     g.permissions = []
     spaces = ProposalSpace.fetch_sorted().filter(ProposalSpace.profile != None).all()
-    return render_template('index.html.jinja2', spaces=spaces)
+    return {'spaces': [d for d in [dict(s.current_access()) for s in spaces] if d]}
+
+
+@funnelapp.route('/', endpoint='index')
+@render_with({'text/html': 'funnelindex.html.jinja2'}, json=True)
+def funnelindex():
+    g.profile = None
+    g.permissions = []
```

Comment:
@jace  `g.permissions` is widely used in this repo. I'm not too sure about the changes required to change them back to `current_auth.permissions`.  Can we handle this in a separate PR?